src/formio/components/Content.stories.mdx

```diff
@@ -0,0 +1,54 @@
+import {Canvas, Meta, Story, ArgsTable} from '@storybook/addon-docs';
+import {FormioComponent, FormDecorator} from './story-util';
+
+<Meta
+  title="Form.io components/Content"
+  argTypes={{
+    id: {
+      type: {name: 'string'},
+    },
+    label: {
+      type: {name: 'string', required: false},
+    },
+  }}
+  decorators={[FormDecorator]}
+  parameters={{
+    controls: {sort: 'requiredFirst'},
+  }}
+/>
+
+# Content
+
+## Default
+
+export const Template = ({customClass = '', html = '', id = '', label = '', ...args}) => (
+  <FormioComponent
+    component={{
+      html,
+      label,
+      key: id,
+      showInEmail: false,
+      showInPDF: true,
+      input: false,
+      tableView: false,
+      customClass,
+      type: 'content',
+    }}
+  />
+);
+
+<Canvas>
+  <Story
+    name="Default"
+    args={{
+      customClass: 'breed',
```

Comment:
this arg will have to be added to the `argTypes` above - the allowed list of values is limited in the backend: https://github.com/open-formulieren/open-forms/blob/master/src/openforms/js/components/form/content.js#L13

You can use the control type of `radio` for this, see https://github.com/open-formulieren/open-forms-sdk/blob/main/src/components/ProgressIndicator/index.stories.mdx?plain=1#L13 for an example